### PR TITLE
Update "es" translation

### PR DESCRIPTION
**Pull request by MELERIX**
also moved the older Spanish Castilian translation to a separate folder and country code to "es_ES".

### CODE REVIEW

**Comment by ICEknigh7** [comment]
I'm seeing that you're swapping some Castilian Spanish terminology for their Latin American counterparts, which is going to raise a lot of eyebrows.

I don't think it's really feasable to have a catch-all "neutral Spanish" translation that extends outside of LA (Halo 2's attempt was very poorly received in Spain), so the best option here might be to do what current streaming services do (for example, [Netflix](https://partnerhelp.netflixstudios.com/hc/en-us/articles/4411631891859-Netflix-Games-Internationalization-Requirements#h_01FYCSEP1ZN37F09WBMGASJ0F9) ) and have two separate "Castilian Spanish" and "Latin American Spanish" translations available.

**Comment by FoxhackDN** [comment]
> I'm seeing that you're swapping some Castilian Spanish terminology for their Latin American counterparts, which is going to raise a lot of eyebrows.

Not only that... some of the terms used here are... questionable. Not exactly very neutral, either.

**Review by angelosa — CHANGES_REQUESTED** [review]
I'm definitely with @ICEknigh7 and @FoxhackDN here, there are several places where the translation is not very neutral.

**Comment by FoxhackDN** [comment]
I also don't think this should replace the existing translation. I think @ICEknigh7's suggestion to create a separate Neutral Spanish one is good.

File: language/Spanish/strings.po [hunk 1]
```diff
@@ -29,26 +29,32 @@ msgid ""
...
+"Este sistema requiere que se monten imágenes de medios para el/los "
+"siguiente(s) dispositivo(s): "
 
 #: src/frontend/mame/ui/ui.cpp:1254
 #, c-format
 msgid ""
 "UI controls enabled\n"
 "Use %1$s to toggle"
 msgstr ""
+"Controles de IU habilitados\n"
```
Comment (angelosa) [inline language/Spanish/strings.po:47]: UI and IU can be used interchangeably in Spanish, I'd leave as UI for consistency.

File: language/Spanish/strings.po [hunk 2]
```diff
@@ -6280,38 +6336,38 @@ msgstr "Hecho"
...
 msgctxt "plugin-timer"
 msgid "%03d:%02d:%02d"
-msgstr ""
+msgstr "%03d:%02d:%02d"
 
 #: plugins/timer/init.lua:32
 msgctxt "plugin-timer"
 msgid "Wall clock"
-msgstr ""
+msgstr "Reloj de pared"
```
Comment (angelosa) [inline language/Spanish/strings.po:6360]: I don't think this translates literally, cfr. https://en.wikipedia.org/wiki/Elapsed_real_time

**Comment by FoxhackDN** [comment]
> any suggestion is welcome, but remember this is main Spanish translation (with literal translation as possible), and not Mexican/Spain/LA translation.

Okay, no. We should NEVER have a literal translation. Is that why you have things like "imagenes de medios" in place of "media images"? And you changed "Gamma" (as in the color parameter) to "Gama", when the word isn't changed in Spanish. Creating a translation isn't just running phrases and words through an automatic translator, there are specific terms that are already in use and we have to make sure to use the right ones.

**Comment by FoxhackDN** [comment]
> > > any suggestion is welcome, but remember this is main Spanish translation (with literal translation as possible), and not Mexican/Spain/LA translation.
> > 
> > 
> > Okay, no. We should NEVER have a literal translation. Is that why you have things like "imagenes de medios" in place of "media images"? And you changed "Gamma" (as in the color parameter) to "Gama", when the word isn't changed in Spanish. Creating a translation isn't just running phrases and words through an automatic translator, there are specific terms that are already in use and we have to make sure to use the right ones.
> 
> media = medios (relativo a medios en informatica u otros usos) aunque si tienes algún otro sinónimo literal coméntalo. gamma = gama (relativo a gama de colores).

"imagen de medio" is the wrong term. The correct term is "imagen de disco" https://es.wikipedia.org/wiki/Imagen_de_disco

"Gamma" in this case is a display term, it's right next to Brightness and Contrast in the strings file. And the common term for that in Spanish is -Gamma- https://support.epson-europe.com/onlineguides/es/cx3200/ref_g/optmz_2.htm

File: language/Spanish/strings.po (hunk 1 above)
Comment (FoxhackDN) [reply language/Spanish/strings.po:47]: UI is the correct term in Spanish, even if the two letters are swapped around.

Comment (angelosa) [reply language/Spanish/strings.po:47]: If you Google Interfaz de Usuario you either get IU or UI, and most often than not it's latter. 
Same for UX btw: it's Experiencia de Usuario but nobody will ever say EU anywhere for obvious reasons ...

**Comment by FoxhackDN** [comment]
> > > > > any suggestion is welcome, but remember this is main Spanish translation (with literal translation as possible), and not Mexican/Spain/LA translation.
> > > > 
> > > > 
> > > > Okay, no. We should NEVER have a literal translation. Is that why you have things like "imagenes de medios" in place of "media images"? And you changed "Gamma" (as in the color parameter) to "Gama", when the word isn't changed in Spanish. Creating a translation isn't just running phrases and words through an automatic translator, there are specific terms that are already in use and we have to make sure to use the right ones.
> > > 
> > > 
> > > media = medios (relativo a medios en informatica u otros usos) aunque si tienes algún otro sinónimo literal coméntalo. gamma = gama (relativo a gama de colores).
> > 
> > 
> > "imagen de medio" is the wrong term. The correct term is "imagen de disco" https://es.wikipedia.org/wiki/Imagen_de_disco
> > "Gamma" in this case is a display term, it's right next to Brightness and Contrast in the strings file. And the common term for that in Spanish is -Gamma- https://support.epson-europe.com/onlineguides/es/cx3200/ref_g/optmz_2.htm
> 
> disk/disc image = imagen de disco gamma = gama https://dle.rae.es/gama (1. f. Escala, gradación de colores.), excepto que te refieras a rayos gamma lo cual no es el caso.

The word of the day is "context". As in "how are these phrases used in the emulator"?

Gamma is a color setting, and it's not changed in Spanish. We're not talking about a color palette here, but a display setting. Why did you even think of "gamma rays"? "Media images" in this case refers to disk images, but your original translation made it clear that you didn't even see how it was used, you just looked at the name and translated it as is.

I don't want to be rude, but you really shouldn't be translating things if you don't have enough experience doing it. And being bilingual isn't enough.

**Comment by ICEknigh7** [comment]
Something I think needs to be pointed out is that the term "Neutral Spanish" usually means neutral to all Latin American countries, something that's achieved by avoiding localisms, etc, but doesn't really work with Castilian Spanish since it has less things in common.

For example, the old translation said "vídeo" , "eliminado" and "bloquear" (which is how it's said in Spain) and it was changed to "video", "removido" and "enforzar" (which is how it's said in America), things like these are why I don't think there's a feasable way to have both language variations in the same translation.

There doesn't even have to be a "main" Spanish translation since MAME defaults to English, anyway. The user could just choose which one of the two they want.

**Pull request closed without merging**